extensions/wikia/AssetsManager/config.php

```diff
@@ -309,6 +309,14 @@
 	],
 ];
 
+$config['adengine2_bridge_js'] = [
```

Comment:
Please explain to me (at your convenience) how groups differ from files. Idk why `adengine2_bridge_js` is a one-file group.

Reply:
I isn't really different. Group with one asset works in the same way as single asset. I left it somewhere in the middle of refactoring and doubled bridge. Fixed :+1: